### PR TITLE
Fix homepage URLs

### PR DESCRIPTION
**Pull request by gavinorland**
Homepages here should have been Github, not npm! Fixed.

### CODE REVIEW

File: components/array-object-table/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "author": "Steve Sims",
   "contributors": ["Gavin Orland"],
   "license": "MIT",
-  "homepage": "https://www.npmjs.com/package/@govuk-frederic/array-object-table",
+  "homepage": "https://github.com/stevesims/govuk-frederic/tree/master/components/array-object-table#readme",
```
Comment (stevesims) [inline components/array-object-table/package.json]: Do these links really need the `#readme` bit?

Comment (gavinorland) [reply components/array-object-table/package.json]: [Apparently](https://docs.npmjs.com/files/package.json#homepage) that's the way it's usually done. Could remove it though if you prefer.

Comment (Loque-) [reply components/array-object-table/package.json]: I don't think they should link to the readme TBH, the package location is perfect as it is the homepage for the component.

Comment (gavinorland) [reply components/array-object-table/package.json]: Okedokie - will search and replace and push this up shortly.

**Pull request merged**